corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -121,10 +142,12 @@ def _migrate_unprocessed_form(self, couch_form_json):
         _migrate_form_operations(sql_form, couch_form)
 
         if couch_form.doc_type != 'SubmissionErrorLog':
-            diffs = json_diff(couch_form.to_json(), sql_form.to_json(), track_list_indices=False)
+            couch_form_json = couch_form.to_json()
+            sql_form_json = sql_form.to_json()
+            diffs = json_diff(couch_form_json, sql_form_json, track_list_indices=False)
             self.diff_db.add_diffs(
                 couch_form.doc_type, couch_form.form_id,
-                filter_form_diffs(couch_form.doc_type, diffs)
+                filter_form_diffs(couch_form_json, sql_form_json, diffs)
             )
```

Comment:
The code in this `if` block is an exact duplicate of [L98-L104](https://github.com/dimagi/commcare-hq/blob/da7846ff59408821c380415163bd68845bbfe693/corehq/apps/couch_sql_migration/couchsqlmigration.py#L98-L104)